roles/pulp-webserver/defaults/main.yml

```diff
@@ -5,3 +5,5 @@ pulp_content_port: 24816
 pulp_api_host: 127.0.0.1
 pulp_api_port: 24817
 pulp_configure_firewall: auto
+pulp_webserver_static_dir: '/var/lib/pulp/assets/'
+pulp_user_home: '/var/lib/pulp'
```

Comment:
1. Needs documentation in this role's README.md about loose coupling / shared variables with the pulp role.

2. `pulp_user_home` was never documented in the pulp role. Use `git blame` to see if there was a reason why it should be undocumented. If no reason is found, document it in the pulp role.